src/NServiceBus.RabbitMQ/AssemblyInfo.cs

```diff
@@ -1,9 +1,3 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
+﻿using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("NServiceBus.RabbitMQ")]
-[assembly: AssemblyCopyright("Copyright NServiceBus. All rights reserved")]
-[assembly: AssemblyProduct("NServiceBus.RabbitMQ")]
```

Comment:
I don't believe those have an impact from a breaking-change standpoint, but then can be viewed in the Details tab if you right-click on the assembly and view properties.

In this PR I was leaning towards trying to keep things as close to how they were before, but we do still need to clean up everything here to align to the component naming guidelines, which would be `NServiceBus.Transport.RabbitMQ`.

It looks like I can add the `Product` property to the csproj, and that wires up the attribute correctly. There is also a `Title` property (which I believe was a late addition based on an issue I recall seeing) that changes the package's `Title` value, but it doesn't appear to be properly hooked up to generating the assembly attribute.

I'll go ahead and add the Title attribute back for now, and in a subsequent PR we can make changes to match the naming conventions.